src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1549299689.inc.php

```diff
@@ -7,13 +7,7 @@
 
 $categoryDef = <<<EOT
 contents             # Contents              # Inhalte               # fas fa-file-alt
-products             # Products & categories # Produkte & Kategorien # fas fa-cubes
```

Comment:
Wouldn't this need to be actually "moved"?

Or do we reset the (test/master) database still?

Reply:
Yes, we need to reset the master database during beta stage.